packages/magento-category/components/CategoryMeta/CategoryMeta.tsx

```diff
@@ -5,17 +5,13 @@ import { i18n } from '@lingui/core'
 import { CategoryMetaFragment } from './CategoryMeta.gql'
 
 export type CategoryMetaProps = CategoryMetaFragment &
-  Omit<PageMetaProps, 'title' | 'metaDescription'> & {
+  Omit<PageMetaProps, | 'children'> & {
     params?: ProductListParams
-    title?: string | null
-    metaDescription?: string | null
```

Comment:
Reverted this change as these types are not compatible with NextPageMetaProps types where title and metaDescription can't be null